### PR TITLE
I14380: Fixed a couple of things in the 'proc_configuring-a-workspace…

### PR DESCRIPTION
**Pull request by supriya-bharadwaj**
…-using-a-devfile.adoc' file.

Signed-off-by: Supriya Takkhi <sbharadw@redhat.com>

### What does this PR do?
Fixes a couple of things in the `src/main/pages/che-7/end-user-guide/proc_configuring-a-workspace-using-a-devfile.adoc` file.

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/14380

@rkratky , can you please have a look? Thanks!

### CODE REVIEW

File: src/main/pages/che-7/end-user-guide/proc_configuring-a-workspace-using-a-devfile.adoc [hunk 1]
```diff
@@ -19,9 +19,7 @@ See the link:https://redhat-developer.github.io/devfile/devfile[Devfile specific
 
 .Prerequisites
 
-* An OKD, OCP, or OSD cluster, if using Che on Openshift. See below for use of link:https://www.okd.io/minishift/[Minishift] (OKD).
-
-* Or, you can use a Kubernetes cluster such as link:https://github.com/kubernetes/minikube#installation[Minikube].
+* An OKD, OCP, or OSD cluster, if using Che on Openshift. See below for use of link:https://www.okd.io/minishift/[Minishift] (OKD). Or, you can use a Kubernetes cluster such as link:https://github.com/kubernetes/minikube#installation[Minikube].
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_configuring-a-workspace-using-a-devfile.adoc]:
```suggestion
* An OKD, OCP, or OSD cluster for using Che on Openshift. See below for use of link:https://www.okd.io/minishift/[Minishift] (OKD). Or, use a Kubernetes cluster, such as link:https://github.com/kubernetes/minikube#installation[Minikube].
```

File: src/main/pages/che-7/end-user-guide/proc_configuring-a-workspace-using-a-devfile.adoc [hunk 2]
```diff
@@ -64,19 +62,19 @@ To create a workspace from a devfile:
 
 . Commit a `devfile.yaml` file into the root of your Git repository.
 
-. When the repository contains a `devfile.yaml` file, point a running Che 7 instance at that link:https://github.com/eclipse/che[project], and it will use the discovered link:https://github.com/eclipse/che/blob/master/devfile.yaml[devfile.yaml] file to build a workspace using the `/f?url=` API.
+. When the repository contains a `devfile.yaml` file, point a running Che 7 instance at that project, and it will use the discovered link:https://github.com/eclipse/che/blob/master/devfile.yaml[devfile.yaml] file to build a workspace using the `/f?url=` API.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_configuring-a-workspace-using-a-devfile.adoc]:
```suggestion
. When the repository contains a `devfile.yaml` file, point a running Che 7 instance at that project. The Che 7 instance then uses the discovered link:https://github.com/eclipse/che/blob/master/devfile.yaml[devfile.yaml] file to build a workspace using the `/f?url=` API.
```

File: src/main/pages/che-7/end-user-guide/proc_configuring-a-workspace-using-a-devfile.adoc [hunk 3]
```diff
@@ -64,19 +62,19 @@ To create a workspace from a devfile:
...
-
++
 image::workspaces/che-in-che-devfile.png[Che in Che]
++
+* Or, point at a plain text file: `++http://che-che.<IP-address>.nip.io/f?url=https://gist.githubusercontent.com/nickboldt/9551284c2c779a93ccf16bc801416205/raw/7ad1e949880d7f8ed30360e74bb9f8426450db05/devfile.yaml++`.
 
-. You can also point at a plain text file: `++http://che-che.<IP-address>.nip.io/f?url=https://gist.githubusercontent.com/nickboldt/9551284c2c779a93ccf16bc801416205/raw/7ad1e949880d7f8ed30360e74bb9f8426450db05/devfile.yaml++`.
-
-. If you are familiar with link:https://github.com/che-incubator/chectl/[chectl], you can also reference a local devfile. For example, to create a workspace in which to build the Che project:
+* If you are familiar with link:https://github.com/che-incubator/chectl/[chectl], reference a local devfile. For example, to create a workspace in which to build the Che project:
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/proc_configuring-a-workspace-using-a-devfile.adoc]:
```suggestion
* Alternatively, use link:https://github.com/che-incubator/chectl/[chectl] to reference a local devfile. For example, to create a workspace in which to build the Che project:
```

**Review by rkratky — DISMISSED** [review]

**Review by themr0c — APPROVED** [review]
LGTM. No errors on che-7 with linkchecker.

**Comment by themr0c** [comment]
fix eclipse/che#14380

**Pull request merged**